### ISSUE TITLE
Default image

### ISSUE THREAD

**Issue by vinzv** [friendica/friendica#7564]
I really don't like the default profile picture. 

![person-80](https://user-images.githubusercontent.com/74432/63655836-5dd3f980-c78d-11e9-9c24-ab518de6e2e1.jpg)

Would it be committing sacrilege to replace it with something more modern or basic? As example:

![image](https://user-images.githubusercontent.com/74432/63656027-a7bddf00-c78f-11e9-9f81-61167e6ae32d.png) 
![image](https://user-images.githubusercontent.com/74432/63656045-cae88e80-c78f-11e9-82a7-f7eb3efcad00.png)

**Comment by nupplaphil** [comment]
I'd vote for the first image, because the meaning of man/woman may also be "group".

**Comment by vinzv** [comment]
Yes, these are just examples for the kind of style. I wanted to show the rough direction before starting to work on some _real_ drafts.

**Comment by MrPetovan** [comment]
Why not using the official mascot?

**Comment by nupplaphil** [comment]
Maybe more in a stilized way, but the mascot sounds nice :)

**Comment by vinzv** [comment]
To be honest: I have no clue what the _official_ mascot is. 🤪

**Comment by vinzv** [comment]
Oh, that one. I did not recognize it as the official mascot. This needs to be pushed way more then.
Using it for default profile pictures would be good then. Any objections?

**Comment by vinzv** [comment]
> It can be an outline of it, or a silhouette.

There you go:

![image](https://user-images.githubusercontent.com/74432/63699815-e0a69400-c821-11e9-9b40-4308be1372be.png) 
![image](https://user-images.githubusercontent.com/74432/63700311-ef417b00-c822-11e9-8604-0e9e06a52d3f.png)
 

> We could push it more, but we have few frontend people and no outreach capabilities.

I'll think about it a bit and collect some ideas separately as this is off topic.

**Comment by annando** [comment]
Would it be possible to have protocol specific default avatars? For example: We do display the default avatar as well when a Mastodon contact hadn't provided an individual one.

**Comment by vinzv** [comment]
> The top one would more appropriate as the goal is to have a legible 50x50 avatar.

Yes. Maybe it needs even thicker lines (pay attention to the upper right menu profile icon).

![image](https://user-images.githubusercontent.com/74432/63702238-dcc94080-c826-11e9-8655-4701c76cf9df.png)

vs.

![image](https://user-images.githubusercontent.com/74432/63702428-3cbfe700-c827-11e9-9c78-c2303ddf977b.png)

**Comment by vinzv** [comment]
> Would it be possible to have protocol specific default avatars?

From graphics side - of course. But this would need to be adapted code sided as well.

**Comment by annando** [comment]
At least I would like to have two different avatars, one for Friendica, one for everything else. This would present the chance to unify this behaviour with a dedicated function in the contact model.
